src/aics-image-viewer/components/App/index.tsx

```diff
@@ -762,97 +763,99 @@ const App: React.FC<AppProps> = (props) => {
   const showControls = { ...defaultShownControls, ...props.showControls };
 
   return (
-    <Layout className="cell-viewer-app" style={{ height: props.appHeight }}>
-      {channelSettings.map((channelState, index) => (
-        <ChannelUpdater
-          key={`${index}_${channelState.name}`}
-          {...{ channelState, index }}
-          view3d={view3d}
-          image={image}
-          version={channelVersions[index]}
-        />
-      ))}
-      <Sider
-        className="control-panel-holder"
-        collapsible={true}
-        defaultCollapsed={false}
-        collapsedWidth={50}
-        trigger={null}
-        collapsed={controlPanelClosed}
-        width={500}
-      >
-        <ControlPanel
-          showControls={showControls}
-          // image state
-          imageName={image?.name}
-          hasImage={!!image}
-          pixelSize={image ? image.imageInfo.physicalPixelSize.toArray() : [1, 1, 1]}
-          channelDataChannels={image?.channels}
-          channelGroupedByType={channelGroupedByType}
-          // user selections
-          channelSettings={channelSettings}
-          showBoundingBox={viewerSettings.showBoundingBox}
-          backgroundColor={viewerSettings.backgroundColor}
-          boundingBoxColor={viewerSettings.boundingBoxColor}
-          maskAlpha={viewerSettings.maskAlpha}
-          brightness={viewerSettings.brightness}
-          density={viewerSettings.density}
-          levels={viewerSettings.levels}
-          interpolationEnabled={viewerSettings.interpolationEnabled}
+    <StyleProvider>
```

Comment:
Recommend turning off whitespace when viewing this file. The only change was to add the `StyleProvider` component around the contents of App, so that it looks the same when embedded or run within the website.

Note that this does mean that `styled-components` is now being used within the viewer component itself.